src/sentry/services/hybrid_cloud/auth/__init__.py

```diff
@@ -150,6 +148,14 @@ class AuthenticationContext:
     auth: AuthenticatedToken | None = None
     user: APIUser | None = None
 
+    def get_user(self):
+        """
+        Helper function to avoid importing AnonymousUser when `applied_to_request` is run on startup
+        """
+        from django.contrib.auth.models import AnonymousUser
+
+        return self.user or AnonymousUser()
```

Comment:
@corps I'm hoping this retains the same behaviour as before but I'm not sure how to test it directly. I just put the import into a subfunction and it doesn't seem to throw an error on startup

Reply:
Ok sure, thank works.  Man I hate the way python typing was built *into the runtime like wut*.